modules/javafx.media/src/main/native/jfxmedia/platform/osx/avf/AVFAudioEqualizer.h

```diff
@@ -148,6 +149,15 @@ class AVFAudioEqualizer : public CAudioEqualizer {
     double *mEQBufferB;
     UInt32 mSampleRate;
     UInt32 mChannels;
+    pthread_mutex_t mBandLock;
+
+    void lockBands() {
+        pthread_mutex_lock(&mBandLock);
```

Comment:
I assume `int pthread_mutex_[lock|unlock]()` returns some kind of error code.  should we check it?